File: VB_NET/Cap04/c04ex01.vb
' c04ex01.vb

Imports System

Module Program
    Sub Main(args As String())
        Dim A As Integer, B As Integer, X As Integer

        Console.Write("Entre o 1o. valor numerico: ")
        A = Integer.Parse(Console.ReadLine())

        Console.Write("Entre o 2o. valor numerico: ")
        B = Integer.Parse(Console.ReadLine())

        X = A + B

        If (X > 10) Then
            Console.WriteLine("Resultado = " & X)
        End If

        Console.Write(vbCrLf & "Aperte <Enter> para encerrar... ")
        Console.ReadLine()
    End Sub
End Module

File: SCALA/Cap04/c04ex01.scala
// c04ex01.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var A, B, X: Int = 0

    print("Entre o 1o. valor numerico: ")
    A = readInt()

    print("Entre o 2o. valor numerico: ")
    B = readInt()

    X = A + B

    if (X > 10) {
      println("Resultado = " + X)
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}
